schema/deploy/shipping/views.sql

```diff
@@ -557,4 +557,104 @@ create or replace view shipping.observation_with_presence_absence_result_v2 as
 comment on view shipping.observation_with_presence_absence_result_v2 is
   'Joined view of shipping.incidence_model_observation_v3 and shipping.presence_absence_result_v1';
 
+
+create or replace view shipping.sample_with_best_available_site_details_v1 as
+
+    with specimen_manifest_site_data as (
+        select distinct on (manifest_id)
+            cast(value->>'sample_id' as integer) as sample_id,
+            trim(both ' ' from document->>'swab_site') as swab_site,
+            trim(both ' ' from document->>'sample_origin') as sample_origin
+        from
+            receiving.manifest,
+            jsonb_array_elements(processing_log)
+        where value->'sample_id' is not null
```

Comment:
Providing a table alias for `jsonb_array_elements` would make it a little more clear where `value` is coming from.

Or, perhaps even more clear, using `jsonb_to_recordset(processing_log) as log(sample_id integer)`, since that both extracts the JSON `sample_id` field, casts it, and assigns a meaningful name.

Reply:
I addressed this in a new commit.